config-store-yaml/src/test/java/io/stargate/config/store/api/yaml/ConfigStoreActivatorYaml.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.stargate.config.store.api.yaml;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+import io.stargate.config.store.api.ConfigStore;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Hashtable;
+import java.util.Objects;
+import org.junit.jupiter.api.Test;
+import org.osgi.framework.BundleContext;
+
+class ConfigStoreActivatorYaml {
```

Comment:
Just wondering do we need to have the `Test` class name suffix for the `surefire` plugin to be able to find it?

Reply:
I did a local run of `./mvnw -f config-store-yaml/ test` and `ConfigStoreYamlTest` executed, but this class did not execute.